sensu-plugins-network-checks.gemspec

```diff
@@ -50,8 +50,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'github-markup',             '~> 1.3'
   s.add_development_dependency 'pry',                       '~> 0.10'
   s.add_development_dependency 'rake',                      '~> 10.0'
+  s.add_development_dependency 'rdoc',                      '~> 4.2.1'
```

Comment:
I assume there is an implicit dependency on rdoc, that may not raise on most developer environments because rdoc is already installed. When I run the test suite on a fresh checkout of this repo within an isolated environment I got an error about a missing rdoc gem:

```bash
stefan@provornyj ~/sensu-plugins-network-checks $ bundle install
Fetching gem metadata from https://rubygems.org/.......
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Using rake 10.4.2
Using i18n 0.7.0
Using json 1.8.3
Using minitest 5.8.3
Using thread_safe 0.3.5
Using tzinfo 1.2.2
Using activesupport 4.2.5
Using ast 2.2.0
Using parser 2.2.3.0
Using astrolabe 1.3.1
Using bundler 1.8.9
Using docile 1.1.5
Using simplecov-html 0.10.0
Using simplecov 0.11.1
Using codeclimate-test-reporter 0.4.8
Using coderay 1.1.0
Using diff-lcs 1.2.5
Using dnsbl-client 1.0.2
Using github-markup 1.4.0
Using method_source 0.8.2
Using mixlib-cli 1.5.0
Using net-ping 1.7.8
Using powerpack 0.1.1
Using slop 3.6.0
Using pry 0.10.3
Using rainbow 2.0.0
Using redcarpet 3.3.4
Using rspec-support 3.4.1
Using rspec-core 3.4.1
Using rspec-expectations 3.4.0
Using rspec-mocks 3.4.1
Using rspec 3.4.0
Using ruby-progressbar 1.7.5
Using rubocop 0.32.1
Using sensu-plugin 1.2.0
Using whois 3.6.3
Using sensu-plugins-network-checks 0.1.2 from source at .
Using yard 0.8.7.6
Bundle complete! 10 Gemfile dependencies, 38 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.

stefan@provornyj ~/sensu-plugins-network-checks $ bundle exec rake default
rake aborted!
LoadError: cannot load such file -- rdoc
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markup/rdoc.rb:2:in `require'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markup/rdoc.rb:2:in `<top (required)>'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markups.rb:2:in `require'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markups.rb:2:in `<module:Markup>'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markup.rb:53:in `instance_eval'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markup.rb:53:in `<module:Markup>'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markup.rb:5:in `<module:GitHub>'
/home/stefan/.gem/ruby/2.0.0/gems/github-markup-1.4.0/lib/github/markup.rb:4:in `<top (required)>'
/home/stefan/sensu-plugins-network-checks/Rakefile:2:in `require'
/home/stefan/sensu-plugins-network-checks/Rakefile:2:in `<top (required)>'
(See full trace by running task with --trace)
```
I could solve this issue and run the test suite successfully by adding the rdoc gem. I also commited this change since I assume this issue may also affect other devs.

Nevertheless, this change is only for smoother development and is not related to the IPv6 feature at all, so if this is going to block the merge, I can remove it from the pull request.